src/panels/config/zone/dialog-zone-detail.ts

```diff
@@ -152,13 +152,13 @@ class DialogZoneDetail extends LitElement {
             {
               name: "latitude",
               required: true,
-              selector: { text: {} },
+              selector: { number: {} },
```

Comment:
Not sure why these were text, I think number makes more sense. 

The problem currently is that the location selector expects its location to be a number, but this was passing in strings. Guess typescript can't catch that?

By passing in strings, it was causing map to glitch out anytime something like `-` was entered in the textfield, like after entiering the first character when starting to type a negative latitude. Number selector handles this better.

Honestly it probably would have made more sense to push these fields to be part of the location selector? That way location selector component would always show both map and lat/lng editor.

Reply:
Agree that including them in location selector makes sense, mainly for accessibility